templates/components/panels/domains.hbs

```diff
@@ -8,7 +8,7 @@
     </header>
 
     <div class="flex-none flex-l flex-row">
-      <p class="flex-grow-1">
+      <p class="flex-grow-1" style="padding-bottom:25px">
```

Comment:
Since I recently found that having some kind of padding attribute in the CSS in general for various sections, I'm wondering if this could be a good thing to put into a very generic `padding` class (for various directions)?